### PR TITLE
feat(prod/cp-deployment/multi-zone): hint for each zone create own cluster

### PR DESCRIPTION
**Pull request by schogges**
I first assumed that more than one zone can be created per cluster. Somewhere I could read between the lines that each individual zone requires an own cluster. So in order to have every page is page 1, I think it would be nice to have.

### CODE REVIEW

File: app/_src/production/cp-deployment/multi-zone.md [hunk 1]
```diff
@@ -185,7 +185,7 @@ Ensure that migrations have been run against the database prior to running the g
 
 ### Set up the zone control planes
 
-You need the following values to pass to each zone control plane setup:
+For every individual zone an own cluster is required. You need the following values to pass to each zone control plane setup:
```
Comment (michaelbeaumont) [inline app/_src/production/cp-deployment/multi-zone.md:188]: I think it should be under the Kubernetes tab

File: app/_src/production/cp-deployment/multi-zone.md [hunk 2]
```diff
@@ -193,7 +193,7 @@ You need the following values to pass to each zone control plane setup:
 {% tabs zone-control-planes useUrlFragment=false %}
 {% tab zone-control-planes Kubernetes %}
 
-On each zone control plane, run:
+For every individual zone create an own cluster and for each run:
```
[the remark this answers is not in the document]
Comment (schogges) [reply app/_src/production/cp-deployment/multi-zone.md:196]: Good point 👍 In this part of the docs there is only some documentation about Universal on k8s in the section for the global CP. Should we add some docs in this section about the zone CPs accordingly?

[the remark this answers is not in the document]
Comment (slonka) [reply app/_src/production/cp-deployment/multi-zone.md:196]: Might be a good idea, please ask other people how they feel about that :)

File: app/_src/production/cp-deployment/multi-zone.md (hunk 1 above)
Comment (jijiechen) [reply app/_src/production/cp-deployment/multi-zone.md]: @schogges if you've made the code change to fix this suggestion, just go ahread and mark this conversaion as "Resolved", don't need to wait for the author to come back and confirm.

File: app/_src/production/cp-deployment/multi-zone.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (schogges) [reply app/_src/production/cp-deployment/multi-zone.md:190]: I have created an [issue](https://github.com/kumahq/kuma-website/issues/2121) to decide if we want to do that 🙂

**Review by slonka — APPROVED** [review]

**Pull request merged**